### ISSUE TITLE
Module configuration

### ISSUE THREAD

**Issue by alfink** [Project-ARTist/meta#22]
Currently, there is no way to configure a module. For example, changing the proxy's IP in a traffic interceptor module requires either a new module or ugly hacks in the module. It would be more comfortable to configure IPs, ports, etc. in Artist Gui. The module could use those values during compilation.

## Affected Projects

- ARTist
- ArtistGui

**Comment by schrnz** [comment]
Agree, the straightforward way might be to include a *configuration* file in the module that declares key-value pairs (of primitive types for starters) that ArtistGui can generate a simple per-module settings activity from and where the configured values are stored in the directory that is accessible to ARTist via the filesystem helper API.

**Comment by sweisgerber-dev** [comment]
+1 for yaml or json based config :)
